ecocodejava.properties

```diff
@@ -1,21 +1,26 @@
 category=External Analysers
 description=Provide a list of static code analyzers (for Java language) to highlight code structures that may have a negative ecological impact: energy and resources over-consumption, "fatware", shortening terminals' lifespan, etc.
 homepageUrl=https://github.com/green-code-initiative/ecoCode
-publicVersions=1.3.0
-archivedVersions=1.2.1
-
+publicVersions=1.4.2
+archivedVersions=1.2.1,1.3.0
 
 defaults.mavenGroupId=io.ecocode
 defaults.mavenArtifactId=ecocode-java-plugin
 
 1.2.1.description=Initial Marketplace publication of plugin for SonarQube 9.9.+ LTS
-1.2.1.sqVersions=[9.9,10.1]
+1.2.1.sqVersions=[9.9,10.3.*]
```

Comment:
This end range didn't need to change. This version should never have been offered in the Marketplace for instances of 10.2.* and 10.3

Reply:
done